guides/source/association_basics.md

```diff
@@ -783,7 +801,8 @@ The following sections give the details of each type of association, including t
 
 ### `belongs_to` Association Reference
 
-The `belongs_to` association creates a one-to-one match with another model. In database terms, this association says that this class contains the foreign key. If the other class contains the foreign key, then you should use `has_one` instead.
+In database terms, the `belongs_to` association says that this class contains the foreign key. This can be used to set up one-to-one, one-to-many or a many-to-many relation, depending on setup.
+If the other class contains the foreign key in a one-to-one relation, then you should use `has_one` instead.
```

Comment:
If the table of the other class contains the reference in a one-to-one relation, then you should use `has_one` instead.